proposed/http-client/http-client.md

```diff
@@ -101,6 +101,7 @@ use Psr\Http\Client\Exception;
 
 /**
  * Thrown when the request cannot be completed because of network issues.
+ * Example when the target host is not available. 
```

Comment:
i would move that to the end of the docblock, and say "For example when the target host name can not be resolved or the connection failed."